src/components/Button/__stories__/Button.docs.mdx

```diff
@@ -0,0 +1,158 @@
+import {Source, Props} from '@storybook/addon-docs';
+import {
+    ButtonExampleViewAction,
```

Comment:
Storybook have [`Story`](https://storybook.js.org/docs/react/writing-docs/docs-page#with-mdx-documentation) component, to insert ordinary stories to MDX

Reply:
We have different design for examples